gst/nnstreamer/tensor_common.c

```diff
@@ -1027,16 +1063,94 @@ gst_tensors_typefind_function (GstTypeFind * tf, gpointer pdata)
   }
 }
 
+static const gchar *gst_tensor_time_sync_mode_string[] = {
+  [SYNC_NOSYNC] = "nosync",
+  [SYNC_SLOWEST] = "slowest",
+  [SYNC_BASEPAD] = "basepad",
+  [SYNC_END] = "error"
```

Comment:
For proper behavior of ```find_key_strv ```, you need to terminate this array with NULL. Add NULL to the last element.

Reply:
Updated to set null when SYNC_END. ```[SYNC_END] = NULL```